cumulusci/tasks/release_notes/task.py

```diff
@@ -25,6 +25,9 @@ class GithubReleaseNotes(BaseGithubTask):
             )
         },
         "publish": {"description": "Publish to GitHub release if True (default=False)"},
+        "include_empty": {
+            "description": "If True, include links to PRs that have no release notes (default=False)"
+        },
```

Comment:
Don't forget to adjust the release_beta flow in cumulusci.yml to set this to True.